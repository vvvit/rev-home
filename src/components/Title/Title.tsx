import * as React from 'react';

import './Title.scss';

export class Title extends React.Component {
    render() {
        return <h1>{this.props.children}</h1>;
    }
}
