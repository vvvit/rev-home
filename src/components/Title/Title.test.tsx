import * as React from 'react';
import { shallow } from 'enzyme';

import { Title } from './Title';

describe('Title', () => {
    it('Должен корректно отрендериться главный заголовок', () => {
        expect(shallow(<Title>Главный заголовок</Title>)).toMatchSnapshot();
    });
});
