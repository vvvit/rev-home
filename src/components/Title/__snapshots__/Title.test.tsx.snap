// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title Должен корректно отрендериться главный заголовок 1`] = `
<h1>
  Главный заголовок
</h1>
`;
